Tests/RxCocoaTests/UITextView+RxTests.swift

```diff
@@ -116,15 +132,26 @@ final class UITextViewTests : RxTest {
 }
 
 final class UITextViewSubclass2 : UITextView {
-    var set: Bool = false
-
+    var settedText = false
+    var settedAttributedText = false
```

Comment:
The same like for `UITextField`. This should probably be `didSetText` and `didSetAttributedText`.